loopin_test.go

```diff
@@ -82,11 +84,17 @@ func TestLoopInSuccess(t *testing.T) {
 		t.Fatal("client subscribing to wrong invoice")
 	}
 
+	typedPreimage, err := lntypes.MakePreimage(testPreimage[:])
+	if err != nil {
+		t.Fatalf("could not make preimage: %v", err)
+	}
+
 	// Server has already paid invoice before spending the htlc. Signal
 	// settled.
 	subscription.Update <- lndclient.InvoiceUpdate{
-		State:   channeldb.ContractSettled,
-		AmtPaid: 49000,
+		State:    channeldb.ContractSettled,
+		AmtPaid:  49000,
+		Preimage: &typedPreimage,
```

Comment:
Do we update this test already now as a preparation for when we implement fast preimage reveal to the client in the future?

Reply:
I'm just always in favor of updating tests with what we expect in real life. Otherwise you get test failures down the line because a field isn't set somewhere. 

Fields aren't needed in this PR, but personally preferred.